lib/modules/versioning/redhat/index.ts

```diff
@@ -20,7 +20,7 @@ class RedhatVersioningApi extends GenericVersioningApi {
 
     const { major, minor, patch, releaseMajor, releaseMinor } = matches;
     const release = [
-      typeof major === 'undefined' ? 0 : Number.parseInt(major, 10),
+      Number.parseInt(major, 10),
```

Comment:
major can't be undefined?

Reply:
no, the regex requires that group
https://github.com/renovatebot/renovate/blob/2da8d70d2026b0a300939c14af8d2c5e035f0997/lib/modules/versioning/redhat/index.ts#L10-L12